src/services/WebAzureSubscriptionProvider.ts

```diff
@@ -103,9 +109,11 @@ class VSCodeAzureSubscriptionProvider extends vscode.Disposable implements Azure
 
     get filters(): AzureSubscription[] {
         const selectedSubscriptionIds = settingUtils.getGlobalSetting<string[] | undefined>('selectedSubscriptions');
-        return this.allSubscriptions
-            .filter(s => selectedSubscriptionIds === undefined || selectedSubscriptionIds.includes(s.subscriptionId))
+        // values may be just subscription ids, or they may be in the (correct) form `${tenantId}/${subscriptionId}`
+        const subscriptions = this.allSubscriptions
```

Comment:
Shouldn't it be fixed by the time `filters` is returned? I don't know if we need to worry about the wrong formatting everywhere in code.